### ISSUE TITLE
Auto-Release No Work-y

### ISSUE THREAD

**Issue by jpoon** [VSCodeVim/Vim#857]
https://travis-ci.org/VSCodeVim/Vim/jobs/165971171

```
Deploying application
Executing prepublish script 'node ./node_modules/vscode/bin/compile -p ./'...
Publishing vscodevim.vim@0.3.1...
Error: Failed Request: Not Found(404) - 
Script failed with status 1
failed to deploy
```

**Comment by jpoon** [comment]
Updated the key, let's see if that works on our next release: https://github.com/VSCodeVim/Vim/commit/ff7e9dbe2f3ac2b0adee3f63fc9e5a7322e311c5

**Comment by jpoon** [comment]
Fixed with e77da66ef27ffe644e8397d590a1eb71cb05ca61
